### ISSUE TITLE
Webpack support

### ISSUE THREAD

**Issue by francescaboe** [rockiger/reactpress#63]
More of a question than an issue: I use webpack as a bundler, and if I just buid the project my react app doesn't show up in the page, are there any configurations I need to twich for it to work?
For context i'm following the [reactpress tutorial](https://rockiger.com/en/reactpress/reactpress-tutorial/) except i created my app with webpack instead of vite.
Thanks!

**Comment by rockiger** [comment]
I would try to masquerade as a create-react-app. This means you need a `/build` folder for your compiled files. And you require a `asset-manifest.json` in that folder.

This should do the trick.

**Comment by BlairCooper** [comment]
@francescaboe , I'm curious if you created a Vite app (per the tutorial) and then added webpack or if you used some other tutorial to form the basis of your app before continuing with the reactpress tutorial?

**Comment by francescaboe** [comment]
> @francescaboe , I'm curious if you created a Vite app (per the tutorial) and then added webpack or if you used some other tutorial to form the basis of your app before continuing with the reactpress tutorial?
> 

Nope, i have a sort of react starter repo with the basic stuff i usually need for a react project and tried to use that one, but in the end I just started a new project straight up with vite (no webpack -i think if you use one you don't need the other) and i couldn't get it to work with wp (:
